src/engine.mjs

```diff
@@ -25,6 +25,10 @@ import { CallSite } from './helpers.mjs';
 import * as messages from './messages.mjs';
 
 export const FEATURES = Object.freeze([
+  {
+    name: 'Object.prototype.__AnnexB__',
```

Comment:
can this just be `'annexB'`

Reply:
I’d rather make it possible to pick and choose which **Annex B** features to include.